data/hai/hai reveal 4 middle.txt

```diff
@@ -235,21 +235,29 @@ mission "Hai Rescue [A04] Tarazed Analysis"
+			`	He stops and looks sheepish for a moment. "Sorry, that was probably way more explanation than you needed. There's a small group of us comms people that have lost people or know people who have, that the authorities can't help. We're scattered all over and we've been looking at this ourselves. Together we identified some ships we thought were suspicious but we've not been able to pin down anything concrete. Bastijn was convinced there was a pattern of interference signatures from the logs; he made the analysis program but it was never definitively distinct from the noise. Now though, well what you've given me is a match!"`
+			`	He casts around for a moment and then plugs in a freshly unpacked data card into his console. "Gimme a minute I can do something for you now. Just uh, sit down for ten minutes and I'll be back with you."`
+			choice
+				`	(Sit down.)`
+				`	(Stand around idly.)`
+			`	After some time, in which he looks like he's installing a series of programs onto it, he returns your original data card and the second one, now loaded up with whatever he was putting on it. "Load this new card into one of your communicator's extra data storage slots, preferably the first one. This new card will run maintenance level processes with the communicator to download the logs from the local station when you enter a system and match the log signatures for ships that are in-system and code up extra info for their ID's. Functionally, they'll show up as 'Smuggler' on your display. You'll have to hunt for them though, cause you'll have to be in the same system as them for any of this to work.`
+			`	"Also uh, maybe keep this quiet. This is..." He clears his throat conspicuously. "Somewhat less than strictly legal, though there's no penalty attached to accessing the data at the level this will do for you. You should focus your search on the uninhabited areas between the Far North and Syndicate space, plus any nearby pirate systems. That's where we've noticed the most suspicious activity."`
```

Comment:
```suggestion
			`	"Also... maybe keep this quiet. This is"-he pauses and clears his throat conspicuously-"somewhat less than strictly legal, though there's no penalty attached to accessing the data at the level this card will use. You should focus your search on the uninhabited areas between the Far North and Syndicate space, plus any nearby pirate systems. That's where we've noticed the most suspicious activity."`
```

Reply:
I do not like this application of grammar, it disagrees with a lot of what I've been taught. Gotta ask if this structure is a common americanism again?

Reply:
Not sure if this is predominantly American or not. Here's a couple references I found/consulted on punctuating interruptions to dialogue:
https://theeditorsblog.net/2018/04/08/when-a-comma-isnt-enough/
https://www.touchstone-editing.com/2017/10/mini-lesson-punctuating-interrupted.html
Though the way you have it might be right. I do prefer my suggestion though, because it feels like including a whole sentence between `This is...` and `somewhat less than` makes that break between the complete thought of "this is less than legal" longer than it should be, if that makes sense.